contractcourt/htlc_incoming_resolver_test.go

```diff
@@ -251,7 +251,7 @@ func (i *incomingResolverTestContext) waitForResult(expectSuccessRes bool) {
 	}
 
 	if !expectSuccessRes {
-		if err != nil {
+		if i.nextResolver != nil {
```

Comment:
does seem like this case is executed, should it be?

Reply:
Not sure if I get what you mean, but in some test cases we don't expect the incoming contest resolver to transform into another resolver. This asserts that.

Reply:
my comment was re: that the assertion changed, but none of the code did. is this a stricter check now that still passes?

Reply:
it appears to be the latter, as if `err != nil` the test would have already failed